core/controllers/ingest.py

```diff
@@ -82,6 +83,19 @@ def clean_data(workbook: dict[str, pd.DataFrame]) -> None:
         worksheet.replace({np.nan: None}, inplace=True)  # replaces np.NAT with None
 
 
+def next_submission_id() -> int:
+    """Get the next submission ID by incrementing the last in the DB.
+
+    :return: The next submission ID.
+    """
+    latest_submission = Submission.query.order_by(Submission.submission_id.desc()).first()
```

Comment:
So, I probably should have told you this, but we were planning on a set specification for the Submission ID. It will be a string in the format "S-R03-1", "S-R03-2"  etc with the Rxx number referring to the round (taken from submission DataFrame). then we can easily, and in human readable format see where the submission was made (and makes it easier to ingest historical data)

Reply:
Updated to this format of `submission_id`